### PR TITLE
Added few ebooks

### PR DESCRIPTION
**Pull request by shrn01**
## What does this PR do?
- Added Essential Algorithms
- Added Production Go
- Added JavaScript Wikibook
- Added Laravel Tips and Tricks
- Updated The Modern Javascript Tutorial to use HTTPS
- Added Essential SQL
- Added TypeScript Handbook

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -809,6 +809,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Let's learn Go!](http://go-book.readthedocs.io/en/latest/)
 * [Practical Cryptography With Go](https://leanpub.com/gocrypto/read) - Kyle Isom
 * [Practical Go Lessons](https://www.practical-go-lessons.com) - Maximilien Andile
+* [Production Go](https://leanpub.com/productiongo) - Herman Schaaf and Shawn Smith (EPUB, HTML, PDF) (:construction: *in process*)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:812]: we add an access note for leanpub - take a look at some of the other titles See also #6852

**Review by eshellman — CHANGES_REQUESTED** [review]
sometimes it's better to link to the web version which links to a pdf. sometimes this is especially true of wikibooks because sometimes they take a lot of time to generate.

Your comments on other PRs have been very helpful.

**Comment by shrn01** [comment]
> sometimes it's better to link to the web version which links to a pdf. sometimes this is especially true of wikibooks because sometimes they take a lot of time to generate.
> 
> Your comments on other PRs have been very helpful.

Updated the link to the wikibook also so the user directly lands into the HTML webpage, where he can navigate to the PDF version and printable version.

**Comment by eshellman** [comment]
you can the 'changes made' label to make sure I notice that a PR has changes to check

**Pull request merged**